try {
    test();
} catch(e) {
    console.log(e);
} finally {
    console.log("done");
}